### ISSUE TITLE
Preprocessing of the data of the Q&A dataset

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#61]
## User story
1. As a developer
2. I want to transform the data of the Q&A dataset into the shape and format that the Gemma Model requires
3. So that  I can perform LLM training afterwards

## Acceptance criteria
* The data of the Q&A dataset is preprocessed so that machine learning can be done afterwards with it
* The data is ready to be trained with the Gemma model

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dominic0df** [comment]
added DoD
